### PR TITLE
Fix stress order in ASE calculator & update reference

### PR DESCRIPTION
**Pull request by YuuuXie**
Adapted from the Pymatgen PR template.

## Summary

Include a summary of major changes in bullet points:

* Fix: the gp stress order in ASE calculator is re-ordered to xx, yy, zz, yz, xz, xy
* Update the mgp reference to the published version

## Checklist

Work-in-progress pull requests are encouraged, but please put [WIP]
in the pull request title.

Before a pull request can be merged, the following items must be checked:

- [x] Code is in the [standard Python style](https://www.python.org/dev/peps/pep-0008/). 
      Run [Black](https://pypi.org/project/black/) on your local machine.
- [x] Docstrings have been added in the [Sphinx docstring format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html). _No doc updated_
- [x] Type annotations are **highly** encouraged. 
- [x] Tests have been added for any new functionality or bug fixes. _Updated_
- [x] All existing tests pass. _Let's see travis_

Note that the CI system will run all the above checks. But it will be much more
efficient if you already fix most errors prior to submitting the PR.

### CODE REVIEW

File: flare/ase/otf_forceonly.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
+"""
```
Comment (jonpvandermause) [inline flare/ase/otf_forceonly.py:1]: Is this module needed? Looks like a lot of duplication of `ase/otf.py`

File: flare/otf_forceonly.py [hunk 2]
```diff
@@ -0,0 +1,550 @@
+import logging
```
Comment (jonpvandermause) [inline flare/otf_forceonly.py:1]: Same for this module -- is it needed?

File: flare/ase/otf_forceonly.py (hunk 1 above)
Comment (YuuuXie) [reply flare/ase/otf_forceonly.py:1]: Oops, thanks for checking that. I guess this file is not supposed to be pushed. I'll remove it.

Comment (YuuuXie) [reply flare/ase/otf_forceonly.py]: fixed via 44763cf

File: flare/otf_forceonly.py (hunk 2 above)
Comment (YuuuXie) [reply flare/otf_forceonly.py]: fixed via 44763cf

File: flare/ase/calculator.py [hunk 3]
```diff
@@ -181,7 +167,7 @@ def calculate_mgp(self, atoms):
                 n, chemenv = ra
                 f, v, vir, e = self.mgp_model.predict(chemenv)
                 self.results["forces"][n] = f
-                self.results["partial_stresses"][n] = vir
+                self.results["partial_stresses"][n] = -vir[:, [0, 3, 5, 4, 2, 1]]
```
[the remark this answers is not in the document]
Comment (YuuuXie) [reply flare/ase/calculator.py]: fixed via 8fed0b1

**Review by jonpvandermause — APPROVED** [review]

**Pull request merged**